rafthttp/http.go

```diff
@@ -214,6 +237,40 @@ func (h *streamHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	<-c.closeNotify()
 }
 
+// snapHandler handles the request for snapshot data.
+// When it receives a request, it parses out the index of target snapshot
+// and retrieves it from snapKeeper. Then it sends snapshot data to
+// the client, and reports send result to raft.
```

Comment:
the sender of the request. i use client because this is a HTTP handler, and client is a known word for it.